sonar-orchestrator/src/main/java/com/sonar/orchestrator/container/Server.java

```diff
@@ -113,8 +113,32 @@ public Version version() {
     return distribution.version();
   }
 
+  /**
+   * @deprecated use {@link #getAppLogs()}, {@link #getWebLogs()}, {@link #getCeLogs()} or {@link #getEsLogs()}
+   */
+  @Deprecated
   public File getLogs() {
-    return FileUtils.getFile(home, "logs", "sonar.log");
+    return getLogFile("sonar.log");
+  }
+
+  public File getAppLogs() {
```

Comment:
Javadoc is missing to explain that behaviour is the same than deprecated `getLogs()` for any version of SonarQube.